src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To do list</title>
+</head>
+<body class="dflex">
+    <div class="listcontainer dflex flexcol">
+        <div class="dflex spacebetween">
+            <div>List</div>
+            <div class="dflex"><img class="icon clearcompleted" src="./img/refreshicon.png" alt="refresh"></div>
```

Comment:
You use the `div` tag with button functionality. It would be better if you use the `button` tag for these elements for semantic reasons, probably you may have to add more styles to those button tags, but it is going to work much better. 😄